tests/quantile.rs

```diff
@@ -36,6 +47,18 @@ fn test_min_skipnan_all_nan() {
     assert!(a.min_skipnan().is_nan());
 }
 
+#[test]
+fn test_argmax() {
```

Comment:
We should test what happens when there is more than one maximum value in the array - you have it of `argmin`, but it's missing for `argmax`.